src/hearings/containers/hearing-actuals/hearing-actuals-view-edit-parties/hearing-actuals-view-edit-parties.component.ts

```diff
@@ -155,14 +154,8 @@ export class HearingActualsViewEditPartiesComponent implements OnInit, OnDestroy
   }
 
   private setUpRoleLists(): void {
-    const plannedParties = this.hearingActualsMainModel.hearingPlanned.plannedHearingDays[this.plannedDayIndex].parties;
     for (const role of this.hearingRoles) {
-      const isPlannedRole = plannedParties.some((plannedParty) => plannedParty.partyRole === role.key);
-      if (isPlannedRole) {
-        this.immutablePartyRoles.push(role);
-      } else {
-        this.mutablePartyRoles.push(role);
-      }
+      this.mutablePartyRoles.push(role);
```

Comment:
@johnbenjamin-hmcts - is `mutablePartyRoles` guaranteed to be truthy at this point?